### PR TITLE
Add Azerbaijani (az) language translations

### PR DESCRIPTION
**Pull request by nasimic**
All packages have been translated to Azerbaijani language (az). As a native speaker there are no major grammatical or spelling errors.

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

File: packages/actions/resources/lang/az/delete.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    ],
+
+    'multiple' => [
+
+        'label' => 'Seçilənləri sil',
+
+        'modal' => [
+
+            'heading' => 'Seçilənləri sil', // When ':label' is used here, the meaning is distorted.
```
Comment (zepfietje) [inline packages/actions/resources/lang/az/delete.php]:
```suggestion
            'heading' => 'Seçilənləri sil',
```
```suggestion
            'heading' => 'Seçilənləri sil', // When ':label' is used here, the meaning is distorted.
```

**Comment by zepfietje** [comment]
Thanks a lot, @nasimic!

Would you also be able to create a PR for FilePond to ensure the file upload also supports Azerbaijani?

Here's the list of supported locales in FilePond: https://github.com/pqina/filepond/tree/master/locale.

Here's where we import them in Filament: https://github.com/filamentphp/filament/blob/8c49d1f398b8c1a1cb2040e173c208e3329ded07/packages/forms/resources/js/components/file-upload.js#L572

**Comment by nasimic** [comment]
You are welcome @zepfietje! 

First of all, you guys are doing a great job bringing this project out. I recently started using filament and I loved it. Of cource I am going to create PR for FilePond translations today.

**Comment by zepfietje** [comment]
Thanks a lot, Nasimi!

After https://github.com/pqina/filepond/pull/937 has been merged, we can add it to Filament too.

**Comment by zepfietje** [comment]
@nasimic, could you handle the import? [filament/packages/forms/resources/js/components/file-upload.js](https://github.com/filamentphp/filament/blob/8c49d1f398b8c1a1cb2040e173c208e3329ded07/packages/forms/resources/js/components/file-upload.js#L572)

**Comment by nasimic** [comment]
@zepfietje Absolutely, looking right now

**Comment by zepfietje** [comment]
Perfect, thanks!

**Comment by nasimic** [comment]
@zepfietje Here you go https://github.com/filamentphp/filament/pull/9427

**Pull request merged**